activation/2.0/_index.md

```diff
@@ -10,29 +10,22 @@ appropriate bean to perform the operation(s).
 
 * [Jakarta Activation 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jaf/releases/2.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Activation 2.0 Specification Document](./activation-spec-2.0-RC3.pdf) (PDF)
-* [Jakarta Activation 2.0 Specification Document](./activation-spec-2.0-RC3.html) (HTML)
+* [Jakarta Activation 2.0 Specification Document](activation-spec-2.0.pdf) (PDF)
```

Comment:
Nit. I would leave the "./" just to be clear on the expectation of the directory location.